figures/helpers.py

```diff
@@ -215,3 +215,21 @@ def period_as_month(month_tuple, fmt='%b-%Y'):
     Returns display date for the given month tuple containing year, month
     """
     return datetime.date(*month_tuple).strftime(fmt)
+
+def get_required_fields_for_user_site(user, site):
+    """
+    Returns only required registration fields from user profile of given site.
```

Comment:
Add `Arguments` in docstring: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

Reply:
Done.